samples/CSharp/SourceGenerators/README.md

```diff
@@ -0,0 +1,35 @@
+🚧 Work In Progress 
+========
+
+These samples are for an in-progress feature of Roslyn. As such they may change or break as the feature is developed, and no level of support is implied.
+
+For more infomation on the Source Generators feature, see the [design document](https://github.com/dotnet/roslyn/blob/master/docs/features/source-generators.md).
+
+Prerequisites
+-----
+
+These samples require Visual Studio 16.6 or higher.
+
+Building the samples
+-----
+Open `SourceGenerators.sln` in Visual Studio or run `dotnet build` from the `\SourceGenerators` directory.
```

Comment:
I don't see `SourceGenerators.sln` in this PR or repo. Did you mean to open `Samples.sln`?

Reply:
Yes, it might help if I added that file to the repo huh?